### PR TITLE
Add simple check to trigger 500 errors

### PR DESCRIPTION
**Pull request by jjmurre**
As a kind of reverse health check.

### CODE REVIEW

File: src/tests/test_health.py [hunk 1]
```diff
@@ -5,3 +5,10 @@
 def test_healthchecks(client):
     response = client.get("/status/health/")
     assert response.status_code == 200
+
+
+@pytest.mark.django_db
+def test_runtime_error(client):
+    with pytest.raises(RuntimeError):
+        response = client.get("/500-test/")
```
Comment (vdboor) [inline src/tests/test_health.py:13]: Of hij raised, of hij assert. Maar niet beide. je kan `api_client.raise_request_exception = False` instellen hiervoor.

Comment (jjmurre) [reply src/tests/test_health.py:13]: Ja, idd, die raise check is voldoende, ik haal de assert weg.

**Review by lbam — APPROVED** [review]

**Comment by lbam** [comment]
Maybe we should do something like

```python
if settings.TESTING:
    urlpatterns.append(path("500-test/", _raise_runtime_error))
```

then override that setting in the test? This can be done with test suite classes: https://docs.djangoproject.com/en/4.0/topics/testing/tools/#overriding-settings. Not sure how it would work with test functions.

**Comment by lbam** [comment]
Oh, too late...

**Pull request merged**